pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -1469,6 +1470,30 @@ default 'raise'
             np.array([self.value], dtype='i8'), tz=self.tz)[0]
         return Timestamp(normalized_value).tz_localize(self.tz)
 
+    def strftime(self, format: str) -> str:
+        """
+        Override datetime.strftime() method so we can display
+        nanosecond precision.
+        """
+        freplace = None  # the string to use for %f
```

Comment:
rather than *always* processing this, can you only do so if 'f' is in the format string.

remove the doc-string as well (as we want the doc-string of the super method)